aspnet/mvc/overview/getting-started/getting-started-with-ef-using-mvc/creating-an-entity-framework-data-model-for-an-asp-net-mvc-application.md

```diff
@@ -33,15 +33,15 @@ This tutorial series explains how to build the Contoso University sample applica
 In this tutorial, you:
 
 > [!div class="checklist"]
-> * Create an MVC web app
-> * Set up the site style
-> * Install Entity Framework 6
-> * Create the data model
-> * Create the database context
-> * Initialize DB with test data
-> * Set up EF 6 to use LocalDB
-> * Create controller and views
-> * View the database
+> * [Create an MVC web app](#Create-an-MVC-web-app)
```

Comment:
Tag names in XHTML are case sensitive.  You need to make all the letters lower case.